packages/video_player/video_player_web/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: video_player_web
+description: Web platform implementation of video_player
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/video_player/video_player_web
+version: 0.0.1
```

Comment:
Prefer `0.1.0` so this package gets a slightly better score on pub

Reply:
Done